obal/data/release_package.yml

```diff
@@ -22,20 +22,17 @@
 - hosts: packages
   serial: 1
   gather_facts: no
-  roles:
-    - git_annex_setup
-    - ensure_package
-    - setup_sources
-    - diff_brew_package
-  tasks:
-    - include: tasks/check_builds.yml
-
+  pre_tasks:
     - name: 'Set tito_releasers'
       set_fact:
-        tito_releasers: "{{ joined_releasers }}"
+        build_package_tito_releasers: "{{ releasers | join(' ')  }}"
```

Comment:
I think we could move this into the role

Reply:
Doing so